packages/amplify_datastore/lib/method_channel_datastore.dart

```diff
@@ -56,23 +55,25 @@ class AmplifyDataStoreMethodChannel extends AmplifyDataStore {
 
   @override
   Future<T> delete<T extends Model>(T model, {QueryPredicate when}) async {
-     try {
+    try {
       var modelJson = model.toJson();
       final Map<dynamic, dynamic> serializedResult =
-      await _channel.invokeMapMethod('delete', <String, dynamic>{
-        'modelName': model.instanceType.modelName(),
+          await _channel.invokeMapMethod('delete', <String, dynamic>{
+        'modelName': model.getInstanceType().modelName(),

```

Comment:
this indentation looks screwy

Reply:
I definitely think you need to rebase- this is using the older delete interface.

Reply:
rebase done